### PR TITLE
docs: update mvux bindable proxy generated name

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md). (for bug fixes / features) 
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/Learn/Mvux/Overview.md [hunk 1]
```diff
@@ -477,8 +477,8 @@ You can then use the MVUX example above as a reference to create your own `IFeed
 
 - Define your own Models
 - MVUX recommends using record types for the Models in your app as they're immutable.
-- The MVUX analyzers auto-generate a bindable proxy for each `partial` `class` or `record` named with a _Model_ suffix.
-- For every public [`IFeed`](xref:Uno.Extensions.Mvux.Feeds) property found in the model, a corresponding property is generated on the bindable proxy.
+- The MVUX analyzers auto-generate a ViewModels for each `partial` `class` or `record` named with a _Model_ suffix.
```
Comment (eriklimakc) [inline doc/Learn/Mvux/Overview.md:480]:
```suggestion
- The MVUX analyzers auto-generate a ViewModel for each `partial` `class` or `record` named with a _Model_ suffix.
```

**Review by eriklimakc — APPROVED** [review]
LGTM

**Comment by agneszitte** [comment]
@ajpinedam, @eriklimakc can these doc updates be backported and released as soon as possible or do we need to wait for the next uno.extensions stable for it?

**Comment by ajpinedam** [comment]
> @ajpinedam, @eriklimakc can these doc updates be backported and released as soon as possible or do we need to wait for the next uno.extensions stable for it?

@agneszitte  We need to wait for the next stable release as these changes are only available on dev versions.

**Comment by agneszitte** [comment]
> > @ajpinedam, @eriklimakc can these doc updates be backported and released as soon as possible or do we need to wait for the next uno.extensions stable for it?
> 
> @agneszitte We need to wait for the next stable release as these changes are only available on dev versions.

Great @ajpinedam, thanks for the confirmation!

**Comment by agneszitte** [comment]
@Mergifyio backport release/stable/5.0

**Pull request merged**